src/utils.ts

```diff
@@ -558,6 +558,20 @@ export function thirtyDaysFromNow(): Date {
   return new Date(Date.now() + THIRTY_DAYS_IN_MILLISECONDS);
 }
 
+/**
+ * Get the current version of Node.js engine from `process.version`.
+ * @return Node.js major release version
+ */
+export function getNodeVersionString(): string {
+  const nodeVersion: RegExpExecArray | null = /\d+/.exec(process.version);
```

Comment:
nit* I think I'd prefer non-regex solution like `process.versions.node.split(".")[0]`

Reply:
Sure, that'll work!